src/main/java/edu/kis/powp/jobs2d/command/loader/LoadCommand.java

```diff
@@ -0,0 +1,69 @@
+package edu.kis.powp.jobs2d.command.loader;
+
+import edu.kis.powp.jobs2d.command.CompoundCommand;
+import edu.kis.powp.jobs2d.command.DriverCommand;
+import edu.kis.powp.jobs2d.command.OperateToCommand;
+import edu.kis.powp.jobs2d.command.SetPositionCommand;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class LoadCommand {
+
+    public static CompoundCommand loadCommandsFromFile(String fileName) {
```

Comment:
Importing tangled with a specified datasource (file). Cannot import for (e.g. URL or text field) reusing this code (without modifying it). No SRP and OCP!

Reply:
Done